### ISSUE TITLE
[Feature]: Suggest new albums based on

### ISSUE THREAD

**Issue by srlamin19** [immich-app/immich#570]
### Feature detail

Would be great if **_Immich_** can suggest a **_smart_** view/creation in **albums** based on **Dates** (_days_, _month_, _year_), **Locations** (_Cities_, _Countries_) in this way of smart views..

### Platform

Web

**Comment by alextran1502** [comment]
Can you elaborate on the feature? It would be nice if you also had some examples.

**Comment by palitu** [comment]
i think it is something like, immich notices that you are typically in place x, then notices that you went to place y, it then says here is a "smart album" which is autmatically generated for your trip to y on date.

**Comment by puregreen59** [comment]
Thought of a great but simiple way of auto creating albums prehaps with a (Needs moderation) label.

Put a page in user settings or admin where you can add dates for birthdays of you and significant others. Even have options for anniversarys and such

You could use this information to auto create simple albums of the day like ' my son, name here 6th birthday (Moderation required)'

It's just I often find auto creation based on content never gets it right, so having it this way could be helpful, and a lot less resource intensive

**Comment by Andrew-71** [comment]
I think it would be a good idea to have a smart album that included photos from a specific source, for example the phone camera.

A use case I can think of and would personally use that for is:
I take a lot of photos on my phone, and generally use it as my main camera. However I also save a ton of meme images. Therefore my gallery is a mess, and whenever I need to find an image from either of these categories I have to go through over 5000 files, with over half of those being completely unrelated. 

In a nutshell, what I'm suggesting is to have an album that sorts photos based on device. For example, if I select "iPhone 12" only show pics shot on an iPhone 12.

**Comment by jrasm91** [comment]
I wonder if you actually need any album features for this, or if it really just searching and filtering. I can imagine a situation where you just search by selecting device:phone and you see that subset.
